GetStream/stream-chat-swift
# [CIS-820] Implement giphy attachmentViewInjector

# What this PR do:
- Implements `giphyAttachmentViewInjector` into application, making giphy attachments available in channel.
# How to test it
- Run the app and send `giphy` command and see if the gif is being sent.


https://user-images.githubusercontent.com/17381941/117784732-27316180-b244-11eb-89b3-ae27b7bd1609.mov


https://user-images.githubusercontent.com/17381941/117784824-3fa17c00-b244-11eb-9daf-4c3ba4ae712a.mov
